### PR TITLE
Boardfrommaster

### PR DESCRIPTION
**Pull request by murple13**
Added Board class and Dictionary class

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/sh4ne/Scrabble_CS3450/pull/37" target="_blank">ReviewNinja

File: Scrabble/Form1.resx [hunk 1]
```diff
@@ -117,4 +117,7 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <metadata name="contextMenuStrip1.TrayLocation" type="System.Drawing.Point, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+    <value>17, 17</value>
+  </metadata>
 </root>
```
Comment (sh4ne) [inline Scrabble/Form1.resx:123]: optional : new line needed at end of file

**Comment by sh4ne** [comment]
:+1: looks good we can fix my optional comment later

**Comment by johnsonjo4531** [comment]
:+1: My only question is which dictionary we are using. I'm guessing we have a plain text one according to the code. We'll have to see how performant it is as just a list, but if it's a problem we'll just fix it later.

**Comment by Peter-Scheelke** [comment]
The code looks good, but it'll need more documentation to pass Stylecop.

As for using a list for the dictionary, you can do a binary search on it, so word checks will only take O(log(n)). I'm not sure how long it'll take to load the dictionary into the list, though.

**Comment by sh4ne** [comment]
There are merge conflicts in this as well.

**Comment by murple13** [comment]
I used the most extensive official scrabble dictionary. It took negligible time to load.

**Comment by joshcv2112** [comment]
:+1: Looks good

**Comment by sh4ne** [comment]
Unit tests will also need to be made for these files, correct?

**Comment by Peter-Scheelke** [comment]
You need to open Scrabble_CS3450\ScrabbleUnitTest\ScrabbleUnitTest.csproj in a text editor, and add the line [anglebracket]Compile Include="FirstThingsFirstUnitTests.cs" / [anglebracket] right below the line that says  [anglebracket]ItemGroup [anglebracket]. That should fix the issue.

Replace the  [anglebracket]s with actual angle brackets. Github is sanitizing their data inputs . . .

**Comment by murple13** [comment]
Is it the first instance of ItemGroup that I put this in?

**Comment by Peter-Scheelke** [comment]
Right above the compile include statements.

**Pull request merged**